pkgs/firehose/lib/src/repo.dart

```diff
@@ -65,6 +66,14 @@ class Repository {
       return '${package.name}-v${package.pubspec.version}';
     }
   }
+
+  Uri calculateReleaseUri(Package package, Github github) {
+    final tag = calculateRepoTag(package);
+    final title = '${package.name} version ${package.pubspec.version}';
```

Comment:
When doing this manually, I always had the title == the tag, just to save me from having to do too much manual work during a release.

I think I'm fine with that convention - there's plenty of info in the tag to help people ID releases.

If we do want to do something different, I'd suggest `package:foo v1.2.3` as it's terse but descriptive.

Reply:
going with `package:foo v1.2.3`, I think its just slightly better than the tag